.github/workflows/deploy-docs.yml

```
@@ -3,11 +3,31 @@ name: Deploy documentation website
+  cancel-in-progress: false
 
 jobs:
-  deploy-pages:
-    uses: it-at-m/.github/.github/workflows/deploy-pages.yml@main
+  deploy-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - id: build_docs
+        uses: it-at-m/.github/.github/actions/action-build-docs@main
+        with:
+          app-path: ${{inputs.app-path}}
```

actionlint (1.7.4)

property "app-path" is not defined in object type {}

(expression)